pkg/util/resize/configspec.go

```diff
@@ -493,3 +522,11 @@ func cmpPtr[T comparable](a *T, b *T, c **T) {
 		*c = b
 	}
 }
+
+func ptrEqual[T comparable](a *T, b *T) bool {
```

Comment:
should be removed if https://github.com/vmware-tanzu/vm-operator/pull/610 goes in first